pkg/cio/io.go

```diff
@@ -254,12 +254,22 @@ func LogURI(uri *url.URL) Creator {
 
 // TerminalLogURI provides the raw logging URI
 // as well as sets the terminal option to true.
-func TerminalLogURI(uri *url.URL) Creator {
+func TerminalLogURI(uri *url.URL, setStderr ...bool) Creator {
 	return func(_ string) (IO, error) {
+		res := uri.String()
+
+		// Default value for Stderr is an ampty string
+		// Windows HCSShim requires that stderr is an empty string when using terminal.
```

Comment:
If the logic here is Windows-specific, should this be factored out into a platform-specific code branch or file?  Is this unconditionally true for Windows and false for non-Windows?

Reply:
> If the logic here is Windows-specific, should this be factored out into a platform-specific code branch or file?

Moved the function to separate files for Windows  and Unix.

>  Is this unconditionally true for Windows and false for non-Windows?

In Windows, when  `setStderr` is explicitly set to `true` or the argument is entirely omitted, the stderr URI is set to the provided URI.

When stderr is explicitly set to false, the stderr URI is set to an empty string.